cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -1108,6 +1108,14 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             }, options));
         },
 
+        getCustomEditModal: function(tabs, editors, xblockInfo, options) {
```

Comment:
Can you please add a docstring to this function? It appears to be unused so someone might delete it without an explanation of why it's here.

Reply:
Sure, done. I've also included the link to an example theme.